docs/swagger/swagger.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
if generated, should probably not be committed !?

Reply:
Well, depends on how you want to look at it, eg. if you want to require everyone to have the whole toolset to run the server - yes. If you want to limit the dev-dependencies to minimum you might want to have this committed. Eg. if you get some testers/FE people to run the server locally, this might just be in the repo. I usually commit such stuff even if it's generated, but I'm open to deleting it as well 😉

Reply:
I think it should be omitted and generated by CI/CD.